.openpublishing.publish.config.json

```diff
@@ -33,6 +33,11 @@
             "path_to_root": "_themes.pdf",
             "url": "https://github.com/Microsoft/templates.docs.msft.pdf",
             "branch": "master"
+        },
+        {  
+            "path_to_root": "Contribute_include",
```

Comment:
Better to place CRR under the docset folder `contribute/` like below
` "path_to_root": "contribute/Contribute_include",`

Also `Contribute_include` value is like an identifier or token for docs-help-pr repo that will be used in the path for includes. Recommend a shorter one may be `cg` or `guide`?
example : ` "path_to_root": "contribute/cg",`

@Jim-Parker to confirm the identifier term to use on crr; whether `cg` or `guide` or `Contribute_include`

<!--StartFragment-->
path_to_root | A relative folder path to repository root.
-- | --
<!--EndFragment-->

Reply:
Let's use "guide". This may cover more than includes as we figure out the sharing model. Thanks